<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>150</width>
      <height>416</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="defaultButton">
       <property name="text">
        <string>Ping Based (default)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="eastusButton">
       <property name="text">
        <string>East US</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="centralusButton">
       <property name="text">
        <string>Central US</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="southcentralusButton">
       <property name="text">
        <string>South Central US</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="westusButton">
       <property name="text">
        <string>West US</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="brazilsouthButton">
       <property name="text">
        <string>Brazil South</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="northeuropeButton">
       <property name="text">
        <string>North Europe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="westeuropeButton">
       <property name="text">
        <string>West Europe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="southafricanorthButton">
       <property name="text">
        <string>South Africa North</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="eastasiaButton">
       <property name="text">
        <string>East Asia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="southeastasiaButton">
       <property name="text">
        <string>South East Asia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="australiaeastButton">
       <property name="text">
        <string>Australia East</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="australiasoutheastButton">
       <property name="text">
        <string>Australia South East</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="japanwestButton">
       <property name="text">
        <string>Japan West</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="changeRegionButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change to region</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings Path: 
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="changePathButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change Path</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
